service_container/tags.rst

```diff
@@ -5,10 +5,15 @@
 How to Work with Service Tags
 =============================
 
-In the same way that a blog post on the web might be tagged with things such
-as "Symfony" or "PHP", services configured in your container can also be
-tagged. In the service container, a tag implies that the service is meant
-to be used for a specific purpose. Take the following example:
+Some container services require to be registered or used in some special way.
+Consider for example the :doc:`Twig extensions </templating/twig_extension>`:
+they are PHP classes that must be added to the main Twig object before using
+them. If you define those exensions as regular Symfony services, Twig will
+ignore them and an exception will be thrown when using them in templates.
```

Comment:
I actually like your *second* paragraph a lot... and I think we should only have it, and remove the first paragraph completely. wdyt?